src/motion/cursor.ts

```diff
@@ -0,0 +1,33 @@
+import * as vscode from 'vscode';
+
+export default class Cursor {
+	editor: vscode.TextEditor;
+
+	constructor(editor: vscode.TextEditor) {
+		this.editor = editor;
+	}
+	
+	moveLeft(times = 1) {
+		for (var index = 0; index < times; index++) {
+			vscode.commands.executeCommand('cursorLeft');
```

Comment:
This loop was super slow in Sublime. We may need to calc the x/y coords of the whole motion and move once. That mean we'd have to handle multiple carets our selves too, which isn't great.